src/Frontend/Themes/Fork/Modules/Blog/Layout/Templates/Detail.html.twig

```diff
@@ -6,53 +6,49 @@
       {{ item.title }}
     </h1>
 
-    <div class="row">
-      <div class="col-xs-12 col-md-6">
+    <div class="d-flex flex-column flex-lg-row justify-content-lg-between">
+      <div>
         {{ 'msg.WrittenBy'|trans|ucfirst|format(item.user_id|usersetting('nickname')) }}
         {{ 'lbl.On'|trans }}
         <time itemprop="datePublished" datetime="{{ item.publish_on|date('Y-m-d\\TH:i:s') }}">
           {{ item.publish_on|spoondate(dateFormatLong, LANGUAGE) }}
         </time>
       </div>
-      <div class="col-xs-12 col-md-6">
-        <div class="pull-right">
-          {% if item.allow_comments %}
-            {% if not item.comments %}
-              <a href="{{ item.full_url }}#{{ 'act.Comment'|trans }}" itemprop="discussionUrl">
+      <div>
+        {% if item.allow_comments %}
+          {% if not item.comments %}
```

Comment:
The content of the if and else are basically the same, can't you write this is a nicer way?